src/controllers/auth.js

```diff
@@ -33,6 +42,28 @@ const createDefaultTables = async () => {
         },
       },
     );
+
+    // fetch all DB tables
+    const tables = tableService.listTables();
+
+    // add permission for the default role (for each db table)
```

Comment:
What if the default roles already exist, e.g. it's not the first that Soul is booting up?

Reply:
Hi @thevahidal, Could you please clarify this question?

Reply:
@AbegaM - the `_users_roles` table may already exist - since Soul may be _re-starting_ rather than _starting-for-the-first-time_.  In which case, the next line `createTable` isn't required.